File: FrontOps/src/app/model/song.ts
export class Song {

    id:number;
    name:string;
  
    constructor (id?:number,name?:string) {
        this.setId(id);
        this.setName(name);
    }
  
    getId() : number {
      return this.id;
    }
    getName() : string {
      return this.name;
    }
  
    setId(id:number) : void {
      this.id = id;
    }
    setName(name:string) : void {
      this.name = name;
    }
  
  }

File: BackOps/src/main/java/com/addcode/company/demo/model/Song.java
package com.addcode.company.demo.model;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.validation.constraints.NotBlank;

@Entity
@Table(name = "songs")
public class Song implements Serializable{
	
	@Id
	@GeneratedValue
	private int id;
	@NotBlank
	private String name;
	
	public Song() {}
	
	public Song(int id, String name) {
		this.setId(id);
		this.setName(name);
	}

	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	
}